lib/libjpeg/jconfigint.h

```diff
@@ -0,0 +1,31 @@
+/* libjpeg-turbo build number */
+#define BUILD  "20191111"
+
+/* Compiler's inline keyword */
+#undef inline
+
+/* How to obtain function inlining. */
+#define INLINE  __inline__ __attribute__((always_inline))
+
+/* Define to the full name of this package. */
+#define PACKAGE_NAME  "libjpeg-turbo"
+
+/* Version number of package */
+#define VERSION  "2.0.4"
+
+/* The size of `size_t', as computed by sizeof. */
+#define SIZEOF_SIZE_T  4
+
+/* Define if your compiler has __builtin_ctzl() and sizeof(unsigned long) == sizeof(size_t). */
+#undef HAVE_BUILTIN_CTZL
```

Comment:
nit: Why is this done? I'd assume this is a potential optimization that we could support?

Reply:
Yes, that should've been enabled.